src/packet/icmpv6.rs.in

```diff
@@ -127,4 +128,697 @@ pub mod Icmpv6Types {
     pub const EchoRequest: Icmpv6Type = Icmpv6Type(128);
     /// ICMPv6 type for "echo reply"
     pub const EchoReply: Icmpv6Type = Icmpv6Type(129);
+    // Neighbor Discovery Protocol [RFC4861]
+    /// ICMPv6 type for "router solicitation"
+    pub const RouterSolicit: Icmpv6Type = Icmpv6Type(133);
+    /// ICMPv6 type for "router advertisement"
+    pub const RouterAdvert: Icmpv6Type = Icmpv6Type(134);
+    /// ICMPv6 type for "neighbor solicitation"
+    pub const NeighborSolicit: Icmpv6Type = Icmpv6Type(135);
+    /// ICMPv6 type for "neighbor advertisement"
+    pub const NeighborAdvert: Icmpv6Type = Icmpv6Type(136);
+    /// ICMPv6 type for "redirect"
+    pub const Redirect: Icmpv6Type = Icmpv6Type(137);
+}
+
+pub mod nd {
```

Comment:
I think this (and the other things below) should be `ndp`, to be consistent with `udp`/`tcp`